WordPressCom-Stats-iOS/Services/WPStatsServiceRemote.m

```diff
@@ -62,11 +66,17 @@ - (instancetype)initWithOAuth2Token:(NSString *)oauth2Token siteId:(NSNumber *)s
         _rfc3339DateFormatter.dateFormat = @"yyyy'-'MM'-'dd'T'HH':'mm':'ssZ";
         _rfc3339DateFormatter.timeZone = [NSTimeZone timeZoneForSecondsFromGMT:0];
 
-        _manager = [AFHTTPRequestOperationManager manager];
+        NSURLSessionConfiguration *sessionConfiguration = [NSURLSessionConfiguration defaultSessionConfiguration];
+        sessionConfiguration.HTTPShouldUsePipelining = YES;
+        sessionConfiguration.HTTPAdditionalHeaders = @{@"Authorization":[NSString stringWithFormat:@"Bearer %@", _oauth2Token]};
+        _manager = [[AFHTTPSessionManager alloc] initWithSessionConfiguration:sessionConfiguration];
+        [_manager setTaskDidCompleteBlock:^(NSURLSession * _Nonnull session, NSURLSessionTask * _Nonnull task, NSError * _Nullable error) {
+            [self updateTaskProgress:session task:task error:error];
```

Comment:
Probably should make this a weak reference to self as the chances of a retain cycle are high. (Static analyzer warning)